Protocol.md

```diff
@@ -732,19 +750,34 @@ RSocket resumption exists only for specific cases. It is not intended to be an 
+As a Requester or Responder __sends__ REQUEST_RESPONSE, REQUEST_FNF, REQUEST_STREAM, REQUEST_CHANNEL, REQUEST_N, CANCEL, ERROR, or PAYLOAD frames, it maintains a __position__ of that frame within an individual stream in that direction if the stream is resumable. In other words __position__ is the number of frames a Requester or Responder has sent withing a single logical stream. The __postion__ value is a 32-bit integer that starts at 0. As a Requester or Responder __receives__ those tracked frames, it maintains an __implied position__ of that frame within the stream in that direction. In other words, __implied position__ is the number of frames received by a Requester or Responder within a single logical stream. The __implied position__ is also a 32-bit value that starts at 0.  
+
+The reason this is “implied” is that the position is not included in each frame and is inferred simply by the message being sent/received within the stream in relation to previous frames in this stream.
 
 This position will be used to identify the location for resuming operation to begin.
 
 Frame types outside REQUEST(s), REQUEST_N, CANCEL, ERROR, and PAYLOAD do not have assigned (nor implied) positions.
 
-When a client sends a RESUME frame, it sends two implied positions: the last frame that was received from the server; the earliest frame position it still retains.  The server can make a determination on whether resumption is possible: have all frames past the client's last-received position been retained? and has the client retained all frames past the server's last-retained position.  If resumption is allowed to continue, the server sends a RESUME_OK frame, indicating its last-received position.
+#### For Connection
+
+As a Server __notifies__ about successfull reception of frames for resumable streams (in the way of sending __implied positions__ for every active resumable stream), the Client MUST maintan a total number of received frames by server based on the provided information. This field is so called __Last Client Available Global Position__. This value is 64-bits long calculated as a sum of all __implied positions__ received from the Server during the whole Client Lifetime.
+
+This position will be used upon receiving a RESUME frame from the Client and aimed to perform a sanity check and identify any possible gups between Client and Server states.
+
+
+Note, all the positions are counted in __number of frames__.
+
+When a client sends a RESUME frame, it sends __Last Client Available Global Position__ and a set of __implied positions__ for active resumable streams.  The server can make a determination on whether resumption is possible: have all frames past the client's last-received position been retained? and has the client retained all frames past the server's last-retained position.  If resumption is allowed to continue, the server sends a RESUME_OK frame, indicating its set of last-received position for active streams.
```

Comment:
Do we want to allow per stream resumption instead? Maybe that is part of the negotation? Rejecting the whole connection because a single stream is lost for an upstream that may have actually disappeared seems wasteful.  And counter to the overall direction of this change.

Reply:
I agree. Will be changing that part